docs/userGuide/reusingContents.md

```diff
@@ -257,6 +264,27 @@ In other words, **`<include>` interprets the reused code relative to the origina
 
 <hr><!-- ======================================================================================================= -->
 
+#### Specifying Variables in the Include Tag
+
+**It is possible to include variables in the include tag.** For example,
```

Comment:
Use `{{ icon_example }}` and `<div class="indented">` whenever an example is listed.